source/documentation/make-updates-during-tax-year.html.md

```diff
@@ -153,11 +153,19 @@ If a customer wants to make a change to a previously submitted periodic update,
 3. Software prepares the summary information and displays it to the customer.
 4. The customer checks the information and submits it.
 5. The software initiates the data submission process by calling one of the following endpoints, depending on the income source type being submitted:
+<<<<<<< HEAD
+   - [Create a Self-Employment Cumulative Period Summary](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-employment-business-api/4.0/oas/page#tag/Self-Employment-Cumulative-Period-Summaries/paths/~1individuals~1business~1self-employment~1%7Bnino%7D~1%7BbusinessId%7D~1period/post)
+   - [Create a UK Property Cumulative Period Summary](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/property-business-api/5.0/oas/page#tag/UK-Property-Cumulative-Period-Summary/paths/~1individuals~1business~1property~1uk~1%7Bnino%7D~1%7BbusinessId%7D~1period~1%7BtaxYear%7D/post)
+   - [Create a Foreign Property Cumulative Period Summary](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/property-business-api/5.0/oas/page#tag/Foreign-Property-Cumulative-Period-Summary/paths/~1individuals~1business~1property~1foreign~1%7Bnino%7D~1%7BbusinessId%7D~1period~1%7BtaxYear%7D/post)
```

Comment:
Please fix link to https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/property-business-api/5.0/oas/page#tag/Foreign-Property-Cumulative-Period-Summary/paths/~1individuals~1business~1property~1foreign~1%7Bnino%7D~1%7BbusinessId%7D~1cumulative~1%7BtaxYear%7D/put and name to `Create and Amend a Foreign Property Cumulative Period Summary`, thanks